tests/ConstantTest.php

```diff
@@ -122,7 +122,7 @@ function defineConstant() {
                     $a = __LINE__;
                     $b = __file__;',
                 'assertions' => [
-                    '$a' => 'int',
+                    '$a' => 'positive-int',
```

Comment:
you'll have to use `int<1, max>` here. positive-int is only used as a read alias, it's not a type Psalm can output